sections/experiments/experiments-chapter.tex

```diff
@@ -500,13 +499,164 @@ \subsubsection*{Ablation study for non-contextual recommendation}
+            (60,0.05409)
+            (80,0.05409)
+            (100,0.05629)
+            (120,0.05585)
+            (140,0.05805)
+            (160,0.06245)
+            (180,0.06201)
+            (200,0.05937)
+        };
+
+        \addplot plot coordinates {
+            (20,0.04266)
+            (40,0.04749)
+            (60,0.04661)
+            (80,0.04925)
+            (100,0.05145)
+            (120,0.04925)
+            (140,0.05277)
+            (160,0.05453)
+            (180,0.05673)
+            (200,0.05717)
+        };
+
+        \addplot plot coordinates {
+            (20,0.04969)
+            (40,0.04705)
+            (60,0.05145)
+            (80,0.05453)
+            (100,0.05585)
+            (120,0.05453)
+            (140,0.05893)
+            (160,0.06069)
+            (180,0.06157)
+            (200,0.06157)
+        };
+
+        \addplot plot coordinates {
+            (20,0.04749)
+            (40,0.04661)
+            (60,0.04749)
+            (80,0.05101)
+            (100,0.05013)
+            (120,0.05057)
+            (140,0.05365)
+            (160,0.05145)
+            (180,0.05277)
+            (200,0.05497)
+        };
+        \legend{Sideinfo + context,Sideinfo,Context,Nothing}
+        \end{axis}
+    \end{tikzpicture}
+    \caption{The performance on HR@20 of CSGCN-IS on the Yelp-NC dataset with different types of input.}
+    \label{fig:ablation_study_context_specific_2}
+\end{figure}
+
+\Cref{fig:ablation_study_context_specific_1} shows the HR@20 for CSGCN-ADJ on YelpNC with different input. 
+We see that when side-information is included in the input for CSGCN-ADJ it does not change the performance much.
+Another interesting thing to note is how the model performs worse when context is used in the input, even though we are doing context-specific recommendations.
+This could be because the CSGCN-ADJ model does not actually model context in way that fits the definition of context defined in \Cref{subsec:define_context_sideinfo}.
+In CSGCN-ADJ, context can instead be viewed as a type of side-information, since it is modelled in a somewhat similar way.
+\\\\
+\Cref{fig:ablation_study_context_specific_2} shows HR@20 for CSGCN-IS on YelpNC with different input.
+For CSGCN-IS we see a slight increase in performance when side-information is included, compared to not having side-information in the input.
+This shows that CSGCN-IS is able to use the side-information to better connect user and item nodes, increasing the collaborative signal.
+We also see that for this model, including context in the input does make a significant different for the performance in a context-specific setting.
+This is because context is modelled with finer granularity than in CSGCN-ADJ, meaning that each item and context value combination is represented by a vector.
+The result of this, is that context can have different effect on the various items, resulting in more accurate context-specific recommendations.
```

Comment:
```suggestion
\Cref{fig:ablation_study_context_specific_1} shows the HR@20 for CSGCN-ADJ on YelpNC with different input. 
We see that when side-information is included in the input for CSGCN-ADJ it does not change the performance much.
Another interesting thing to note is how the model performs worse when the context is used in the input, even though we are doing context-specific recommendations.
This could be because the CSGCN-ADJ model does not model context in a way that fits the definition of context defined in \Cref{subsec:define_context_sideinfo}.
In CSGCN-ADJ, context can instead be viewed as a type of side-information, since it is modeled in a somewhat similar way.
\\\\
\Cref{fig:ablation_study_context_specific_2} shows HR@20 for CSGCN-IS on YelpNC with different input.
For CSGCN-IS we see a slight increase in performance when side-information is included, compared to not having side-information in the input.
This shows that CSGCN-IS is able to use the side-information to better connect user and item nodes, increasing the collaborative signal.
We also see that for this model, including the context in the input does make a significant difference for the performance in a context-specific setting.
This is because context is modeled with finer granularity than in CSGCN-ADJ, meaning that each item and context value combination is represented by a vector.
The result of this is that context can have a different effect on the various items, resulting in more accurate context-specific recommendations.
```